wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -226,20 +227,19 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
     private func createCellDescriptions(in context: ConversationMessageContext) {
         cellDescriptions.removeAll()
 
-        let isSenderVisible = self.isSenderVisible(in: context) && message.senderUser != nil
+        let isSenderVisible = self.shouldShowSenderDetails(in: context) && message.senderUser != nil
```

Comment:
```suggestion
        let shouldShowSenderDetails = self.shouldShowSenderDetails(in: context)
```
You don't need to check if the sender is not nil because that's already done in `shouldShowSenderDetails`

Reply:
Done!